app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java

```diff
@@ -294,7 +294,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     }
 
     @Override
-    public void onViewCreated(View view, Bundle savedInstanceState) {
+    public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
```

Comment:
Why did you add NonNull for the first parameter but did nothing for the second parameter?